lib/std/start.zig

```diff
@@ -300,13 +300,51 @@ fn _start() callconv(.Naked) noreturn {
             \\ and sp, x0, #-16
             \\ b %[posixCallMainAndExit]
             ,
+            .arc =>
+            // The `arc` tag currently means ARCv2, which has an unusually low stack alignment
+            // requirement. ARCv3 increases it from 4 to 16, but we don't support ARCv3 yet.
```

Comment:
https://raw.githubusercontent.com/wiki/foss-for-synopsys-dwc-arc-processors/toolchain/files/ARCv2_ABI.pdf